bundles/org.openhab.ui.dashboard/README.md

```diff
@@ -0,0 +1,34 @@
+# Dashboard tiles
+
+OpenHAB dashboard is landing page for the user where all openHAB UI's can be found. Dashboard support also links to external services. Links can be added to dashboard by ```conf/services/dashboard.cfg``` configuration file.
+
+## Link Configuration
+
+| Parameter name  | Type    | Description                                                                             |
+|-----------------|---------|-----------------------------------------------------------------------------------------|
+| link-nameX      | String  | Name which is shown in the openHAB dashboard.                                           |
+| link-urlX       | String  | URL to external service.                                                                |
+| link-overlayX   | String  | Image overlay icon. Supported values are empty (no icon), "html5", "android" or "apple" |
```

Comment:
Actually many more. If I remember right, all icons from https://icomoon.io/#preview-free should work as an overlay.